<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snort Rule Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="lineColumn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>71</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="hexColumn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>421</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textColumn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>221</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="actionCombo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>81</width>
      <height>33</height>
     </rect>
    </property>
    <property name="currentText">
     <string>alert</string>
    </property>
    <item>
     <property name="text">
      <string>alert</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>log</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>pass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>activate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dynamic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>drop</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>reject</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sdrop</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="protoCombo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>81</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>tcp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>udp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>icmp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ip</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="srcCombo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>any</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="srcPortCombo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>81</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>any</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="dirCombo">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>61</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt;-&gt;</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="destCombo">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>any</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="destPortCombo">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>81</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>any</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="ruleText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>503</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="packetBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>53</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Packet</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
